ocp-resources/compliance-operator-rosa-subscription.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: compliance-operator-sub
+  namespace: openshift-compliance
+spec:
+  channel: alpha
+  name: compliance-operator
+  source: compliance-operator
+  sourceNamespace: openshift-marketplace
+  config:
+    nodeSelector:
+      node-role.kubernetes.io/worker: ""
```

Comment:
@rhmdnd Should this also set an env variable  `PLATFORM` to `ROSA`?

Reply:
Good question - we shouldn't need that any more since we're detecting ROSA environments automatically through the API:

https://github.com/ComplianceAsCode/compliance-operator/pull/518/commits/66ef73d14c9649231d146a48f005fd4ba43f504f